scripts/utils.js

```diff
@@ -19,7 +19,7 @@ export const [setLibs, getLibs] = (() => {
     (prodLibs, location) => {
       libs = (() => {
         const { hostname, search } = location || window.location;
-        if (!(hostname.includes('.hlx.') || hostname.includes('local'))) return prodLibs;
+        if (!['.hlx.', '.stage.', 'local'].some((i) => hostname.includes(i))) return prodLibs;
```

Comment:
Any reason for not going for the more specific 'localhost' hostname?

Reply:
No reason, I wanted to use the same string as was used in the former version